### PR TITLE
March batch

### PR DESCRIPTION
**Pull request by fm**
Adding a couple more companies + updating some

### CODE REVIEW

File: companies/zoho.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        "Zoho Social",
+        "Zoho Survey",
+        "Zoho Vault",
+        "Zoho Writer",
+        "ManageEngine",
+        "Site24x7",
+        "WebNMX"
+    ],
+    "address": "Beneluxlaan 4B\n3527 HT Utrecht\nThe Netherlands",
+    "phone": "+31 85 066 6700",
```
Comment (mal-tee) [inline companies/zoho.json:42]: I can't find a reference to this number in the sources, did I miss sth.? :D

**Review by mal-tee — COMMENTED** [review]
Thank you very much!

File: companies/cloudapp.json [hunk 2]
```diff
@@ -7,14 +7,21 @@
     "runs": [
         "CloudApp (www.getcloudapp.com)"
     ],
-    "address": "PO Box 29502 #96560\nLas Vegas\nNV 89126-9502\nUnited States of America",
-    "email": "privacy@getcloudapp.com",
+    "address": "431 Tehama St\nSan Francisco CA 94103\nUnited States of America",
+    "email": "compliance@getcloudapp.com",
```
[the remark this answers is not in the document]
Comment (baltpeter) [reply companies/cloudapp.json:11]: In cases like this, it's really just a judgement call. I'd say it's fine but I'll leave that to you.

**Review by mal-tee — APPROVED** [review]

File: companies/zoho.json (hunk 1 above)
Comment (fm) [reply companies/zoho.json:42]: https://www.zoho.com/contactus.html

**Review by mal-tee — APPROVED** [review]
Thanks again!

**Pull request merged**